examples/ping_pong/Makefile

```diff
@@ -2,11 +2,11 @@ PROJECT = ping_pong
 
 DEPS = cowboy lasse sync
 
-dep_cowboy = https://github.com/extend/cowboy.git 0.10.0
-dep_lasse = ../../ master
-dep_sync = https://github.com/rustyio/sync master
+dep_cowboy = git https://github.com/ninenines/cowboy.git  2.0.0-pre.3
+dep_lasse = git https://github.com/inaka/lasse.git davecaos.cowboy2x
```

Comment:
Please replace this branch by `cowboy2`. Even if it doesn't work now, it will work once this PR is merged.
`davecaos.cowboy2x` branch will probably be deleted soon.